Source/Diagnostics/ReducedDiags/Costs.H

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2019-2020 Yinjian Zhao
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef WARPX_DIAGNOSTICS_REDUCEDDIAGS_COSTS_H_
+#define WARPX_DIAGNOSTICS_REDUCEDDIAGS_COSTS_H_
+
+#include "WarpX.H"
+#include "ReducedDiags.H"
+
+/**
+ *  This class mainly contains a function that update the
+ *  costs (used in load balance) for writing to output.
+ */
+class Costs : public ReducedDiags
+{
+public:
+
+    /** number of data fields we save for each box
+     *  (cost, processor, level, i_low, j_low, k_low) */
+    const int nDataFields = 6;
```

Comment:
For member variables, it's better to add a "m_" at the beginning, like `m_nDataFields`, `m_nBoxesMax`.

Reply:
Updated.